data/json/recipes/food/frozen.json

```diff
@@ -0,0 +1,247 @@
+[
+  {
+    "result": "milkshake",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 10000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "components": [
+      [ [ "milk", 1 ], [ "con_milk", 1 ], [ "can_coconut", 1 ] ],
+      [
+        [ "chocolate", 1 ],
+        [ "candy", 1 ],
+        [ "candy2", 1 ],
+        [ "candy3", 1 ],
+        [ "maple_candy", 1 ],
+        [ "syrup", 1 ],
+        [ "strawberries", 1 ],
+        [ "strawberries_cooked", 1 ],
+        [ "irradiated_strawberries", 1 ],
+        [ "blueberries", 1 ],
+        [ "blueberries_cooked", 1 ],
+        [ "irradiated_blueberries", 1 ],
+        [ "blackberries", 1 ],
+        [ "irradiated_blackberries", 1 ],
+        [ "cherries", 1 ],
+        [ "irradiated_cherries", 1 ]
+      ],
+      [ [ "sugar", 10 ] ],
+      [ [ "water_clean", 1 ] ]
```

Comment:
Milkshakes are usually made from milk + pre-made ice cream, not milk + ice cream ingredients.
It should also require a food processor, quern, or mortar for grinding the ice cream into the milk.

Reply:
No, ice cream is fairly soft, querns and mortars are for turning hard dry stuff into dust.